pkg/backend/display/json.go

```diff
@@ -176,7 +176,7 @@ func ShowPreviewDigest(events <-chan engine.Event, done chan<- bool, opts Option
 
 				if m.Old != nil {
 					oldState := stateForJSONOutput(m.Old.State, opts)
-					res, err := stack.SerializeResource(oldState, config.NewPanicCrypter(), false /* showSecrets */)
+					res, err := stack.SerializeResource(oldState, config.NewPanicCrypter(), false /* showSecrets */, nil /* aliases */)
```

Comment:
Is this `nil` allright? It's under `display/show` so not as worried semantically but will it change how things are displayed?